doc/source/examples/dem/rectangular-hopper/rectangular-hopper.rst

```diff
@@ -234,6 +235,117 @@ The simulated mass discharging rate is 84.94 g/s.
+        end
+    end
+
+Lagrangian physical properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The total number of particles of this simulation is 6790: 6 times less than the previous example.
+
+.. code-block:: text
+
+    subsection lagrangian physical properties
+        set gx                       = 0.0
+        set gy                       = -9.81
+        set gz                       = 0.0
+        set number of particle types = 1
+        subsection particle type 0
+            set size distribution type            = uniform
+            set diameter                          = 0.00224
+            set number                            = 6790
+            set density particles                 = 2500
+            set young modulus particles           = 1e6
+            set poisson ratio particles           = 0.3
+            set restitution coefficient particles = 0.94
+            set friction coefficient particles    = 0.2
+            set rolling friction particles        = 0.09
+        end
+        set young modulus wall           = 1e6
+        set poisson ratio wall           = 0.3
+        set friction coefficient wall    = 0.2
+        set restitution coefficient wall = 0.9
+        set rolling friction wall        = 0.09
+    end
+
+Insertion info
+~~~~~~~~~~~~~~
+
+Since the geometry of the mesh and the number of the particles are not the same, the insertion info have to be modified according to the new domain of the mesh with an inserted number of particles corresponding to the new number.
+
+.. code-block:: text
+
+    subsection insertion info
+        set insertion method                               = non_uniform
+        set inserted number of particles at each time step = 2910
+        set insertion frequency                            = 25000
+        set insertion box minimum x                        = -0.1030
+        set insertion box minimum y                        Rectangular hopper at the end of the simulation.   = 0.10644
+        set insertion box minimum z                        = .00224
+        set insertion box maximum x                        = 0.1030
+        set insertion box maximum y                        = 0.16020
+        set insertion box maximum z                        = 0.03136
+        set insertion distance threshold                   = 1.5
+        set insertion random number range                  = 0.1
+        set insertion random number seed                   = 20
+    end
+
+Results comparison
+~~~~~~~~~~~~~~~~~~
+Here is the comparison of the results from the original simulation with Lethe DEM, the simulation with periodic boundary conditions with Lethe and the results from Anand et al. paper.
+The simulated mass discharging rate is 84.94 g/s from the original simulation and 88.77 g/s with PBC.
+
+.. figure:: images/figure_hopper_comparison.png
+    :width: 600
+    :alt: Comparison
+    :align: center
+
+    Comparison of mass discharge results from the 2 simulations and the paper.
+
```

Comment:
Can you maybe add a comparison of the run time? I presume the periodic case has a much lower run time no?

Reply:
Yes from about 3h to 30 min